src/server/scripts/Northrend/Ulduar/Ulduar/boss_xt002.cpp

```diff
@@ -499,14 +499,14 @@ struct npc_pummeller : public ScriptedAI
         me->SetReactState(REACT_PASSIVE);
         _scheduler.CancelAll();
 
-        if (_instance->GetBossState(BOSS_XT002) != IN_PROGRESS)
+        if (_instance->GetBossState(DATA_XT002) != IN_PROGRESS)
         {
             me->DespawnOrUnsummon();
             return;
         }
 
 
```

Comment:
One of these 2 blank lines are likely part of the file conflict (in case you wondered).